src/cmd_line/commands/file.ts

```diff
@@ -3,7 +3,7 @@ import * as node from '../node';
 import * as path from 'path';
 import * as util from 'util';
 import * as vscode from 'vscode';
-import untildify = require('untildify');
+import * as untildify from 'untildify';
```

Comment:
in esm output `import = require` is being treated as non valid (only allowed to import types) and stripped out.

![image](https://user-images.githubusercontent.com/1210596/64557888-684ad300-d2f7-11e9-91f8-0427362be47e.png)